erigon-lib/kv/tables.go

```diff
@@ -368,6 +368,9 @@ const (
 	BorCheckpoints    = "BorCheckpoints"            // checkpoint_id -> checkpoint (in JSON encoding)
 	BorCheckpointEnds = "BorCheckpointEnds"         // start block_num -> checkpoint_id (first block of checkpoint)
 
+	// Polygon Bridge
+	PolygonBridge = "PolygonBridge" // bridge event.ID -> event
```

Comment:
it will be good to call this "PolygonBridgeEvents"(both var name and string val) - following "BorEvents" which we are replacing
or an alternative is to not add this and re-use "BorEvents" as the table given that this component works on its own separate DB

Reply:
Renaming to PolygonBridgeEvents for now, we can switch to BorEvents once we have tests and everything works fine.